npm/src/index.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+// @ts-check
+
+import { spawnSync } from "child_process"
+import { fileURLToPath } from "url"
+import { dirname } from "path"
+import EasyDl from "easydl"
```

Comment:
Instead of downloading the dependencies, we should use the binaries that are produced locally.

Reply:
So we'll compile the binary on user's machine?

Reply:
If you're talking about node dependencies all of them are included in node except `easydl` which is a zero dependency library.

Reply:
Yeah, we should compile them on the user's machine. In fact we have a job that does this already and uploads to github releases. We can use those builds directly. That way `easydl` might not be required.

Reply:
Sorry a little confused here 
> We can use those builds directly

Isn't that what the EasyDL code does pull from GitHub releases API